benchmarks/pipe.py

```diff
@@ -681,14 +590,18 @@ def bench_mpi(args):
 parser.add_argument(
     "--no-pipelined-backward", dest="pipelined_backward", action="store_false", help="Pipelined backward pass"
 )
+parser.add_argument("--use_synthetic_data", default=True, help="Uses synthetic data to run benchmarks.")
+parser.add_argument(
+    "--model_name", default="seq_pred", choices=["seq_pred", "transformer"], help="Model used to benchmark pipe."
+)
 parser.set_defaults(pipelined_backward=True)
 
 if __name__ == "__main__":
     args = parser.parse_args()
-    # bench_multi_process(args, all_at_once=True)
+    # FIXME: Add support for multiprocess benchmarking.
```

Comment:
use TODO (name) so that we can know the context of this TODO in the future.

Reply:
Done.